interfaces/cython/cantera/reaction.pyx

```diff
@@ -422,7 +418,7 @@ cdef class Falloff:
     :param init:
         Used internally when wrapping :ct:`Falloff` objects returned from C++.
     """
-    falloff_type = SIMPLE_FALLOFF
+    falloff_type = "Falloff"
```

Comment:
I think this name needs to correspond to the name used by `newFalloff`.

Reply:
Good catch - I was thinking about the class name.